src/utils/DatabaseUtilsTrait.php

```diff
@@ -30,7 +30,7 @@ protected function validateDbConnection(): void
      *
      * @return array|null
      */
-    protected function getDatabase(string $database): ?string
+    protected function getDatabase(?string $database): ?string
```

Comment:
We should see which other changes must be done, the base generators are in the old PHP style so maybe a good refactoring must be done.